src/components/SideBar.js

```diff
@@ -0,0 +1,249 @@
+import { Stack } from './publiq-ui/Stack';
+import { Link } from './publiq-ui/Link';
+import { List } from './publiq-ui/List';
+import { useTranslation } from 'react-i18next';
+import { Icon, Icons } from './publiq-ui/Icon';
+import { getValueFromTheme } from './publiq-ui/theme';
+import { ListItem } from './publiq-ui/ListItem';
+import { Box } from './publiq-ui/Box';
+import { Title } from './publiq-ui/Title';
+import { Button } from './publiq-ui/Button';
+
+const getValueForMenuItemLink = getValueFromTheme('menuItemLink');
+const getValueForMenuItemButton = getValueFromTheme('menuItemButton');
+const getValueForSideBar = getValueFromTheme('sideBar');
+const getValueForMenu = getValueFromTheme('menu');
+
+const MenuItem = ({ href, iconName, children, onClick }) => {
+  const Content = ({ iconName, children }) => (
+    <>
+      <Icon name={iconName} marginRight={3} />
+      <Box as="span">{children}</Box>
+    </>
+  );
+
+  Content.propTypes = {
+    iconName: PropTypes.string,
+    children: PropTypes.node,
+  };
+
+  return (
+    <ListItem>
+      {href ? (
+        <Link
+          href={href}
+          css={`
+            display: flex;
+            width: 100%;
+            text-decoration: none;
+            color: ${getValueForMenuItemLink('color')};
+            padding: 4px;
+
+            &:hover {
+              color: ${getValueForMenuItemLink('hover.color')};
+              background-color: ${getValueForMenuItemLink(
+                'hover.backgroundColor',
+              )};
+              text-decoration: none;
+            }
+          `}
+        >
+          <Content iconName={iconName}>{children}</Content>
+        </Link>
+      ) : (
+        <Button
+          css={`
+            &.btn-primary {
+              display: flex;
+              width: 100%;
+              text-decoration: none;
+              padding: 4px;
+              background-color: inherit;
+              border-color: ${getValueForMenuItemButton('borderColor')};
+
+              &:hover {
+                background-color: ${getValueForMenuItemButton(
+                  'hover.backgroundColor',
+                )};
+                text-decoration: none;
+                border-color: ${getValueForMenuItemButton(
+                  'hover.backgroundColor',
+                )};
+              }
+
+              // active & focus
+              &:not(:disabled):not(.disabled):active:focus,
+              &:not(:disabled):not(.disabled).active:focus,
+              .show > &.dropdown-toggle:focus {
+                background-color: ${getValueForMenuItemButton(
+                  'hover.backgroundColor',
+                )};
+                border-color: ${getValueForMenuItemButton(
+                  'hover.backgroundColor',
+                )};
+              }
+            }
+          `}
+          onClick={onClick}
+        >
+          <Content iconName={iconName}>{children}</Content>
+        </Button>
+      )}
```

Comment:
Not sure if it makes sense to change MenuItem for 1 scenario. a completely custom menu for the notifications would make more sense imo